Application/Client/src/app/app.module.ts

```diff
@@ -28,8 +28,7 @@ import {mapData} from  './mapData/mapData';
 import { FilterComponent } from './filter/components/filter.component';
 import { SearchComponent } from './_common/search/components/search.component';
 import { LoaderComponent } from './_common/loader/components/loader.component';
-
-
+import { SelectInterestComponent } from './_common/select-interest/components/select-interest.component';
```

Comment:
This import is wrong use only select-interest and introduce index.ts file in component